doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -65,32 +65,19 @@ See :ref:`building_coq_project`.
    To compile `foo.v` as part of a module `Mod1` that is rooted
    at `.` (i.e. the directory containing `foo.v`), run `coqc -Q . Mod1 foo.v`.
 
-   To make the module available in `CoqIDE`, include the following line in the
-   `_CoqProject` file (see :ref:`coq_makefile`) in the directory from which you
-   start `CoqIDE`.  *<PATH>* is the pathname of the directory containing the module,
-   which can be an absolute path or relative to Coq's current directory.  For now,
-   you must close and reload a named script file for `CoqIDE` to pick up the change,
-   or restart `CoqIDE`.
-   The project file name is configurable in `Edit / Preferences / Project`.
+   To make the module available in `CoqIDE`, include the following
+   line in the `_CoqProject` file (see :ref:`coq_makefile`) in the
+   directory from which you start `CoqIDE` or give it as an argument
+   to the ``coqide`` command. *<PATH>* is the pathname of the
+   directory containing the module, which can be an absolute path or
+   relative to Coq's current directory. For now, you must close and
+   reload a named script file for `CoqIDE` to pick up the change, or
+   restart `CoqIDE`. The project file name is configurable in `Edit /
+   Preferences / Project`.
 
       .. coqdoc::
          -R <PATH> Mod1
 
-   It's also possible to load a module within `coqtop` or `coqide` with
-   commands like these.  The drawback of this is that it adds
-   environment-specific information (the PATH) to your script, making
-   it non-portable, so we discourage using this approach.
```

Comment:
The text already says "we discourage using this approach" and `Add LoadPath` is deprecated (removal subject to no significant objections).  In the meantime, it's still a feature.  Let's remove this only if and when `Add LoadPath` is removed.

Reply:
This sort of example is meant to show good practice not random features

Reply:
Have we resolved this in the `Add LoadPath` description (your words) and decided to drop the command?  If not, it's premature to drop this.  If we keep the command we should keep this example.

"If [Add LoadPath] is an important feature for you, please open an issue at https://github.com/coq/coq/issues and explain your workflow."

The beginning of this chapter is poorly organized and the entire example ("Compiling and loading a single file") probably should go elsewhere, likely into the Configuration Basics section I'm actively working on.  I will probably reorganize the beginning of this chapter (up to "By command line options") as part of that PR.

Reply:
Whether we are removing it is irrelevant to whether it should be recommended by putting it in random examples